package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@constructor-io/constructorio-ui-autocomplete",
-  "version": "1.9.2",
```

Comment:
The diff seems to have broken somehow and it thinks that upstream main is on 1.9.2... Should not be any problem when we merge it since its actually on 1.9.3 currently.